client/blocks/reader-featured-image/index.jsx

```diff
@@ -2,56 +2,69 @@ import classnames from 'classnames';
 import PropTypes from 'prop-types';
 import cssSafeUrl from 'calypso/lib/css-safe-url';
 import resizeImageUrl from 'calypso/lib/resize-image-url';
-
+import {
+	READER_FEATURED_MAX_IMAGE_HEIGHT,
+	READER_CONTENT_WIDTH,
+} from 'calypso/state/reader/posts/sizes';
 import './style.scss';
 
 const noop = () => {};
 
 const ReaderFeaturedImage = ( {
-	imageUrl,
-	imageWidth,
+	canonicalMedia,
 	href,
 	children,
 	onClick,
 	className,
 	fetched,
+	imageUrl,
+	imageWidth,
+	imageHeight,
 } ) => {
-	if ( imageUrl === undefined ) {
+	const featuredImageUrl = imageUrl || canonicalMedia.src;
```

Comment:
nit: I think I'd do a null safe check for `canonicalMedia.src` I see the prop is _required_ but I don't think that is really enforced.
```suggestion
	const featuredImageUrl = imageUrl || canonicalMedia?.src;
```

Reply:
Good idea, I'll add this.

Reply:
Good idea, added null safe check